sdk/python/kfp/_client.py

```diff
@@ -99,10 +99,11 @@ def __init__(self, host=None, client_id=None, namespace='kubeflow', other_client
       other_client_id: The client ID used to obtain the auth codes and refresh tokens.
         Reference: https://cloud.google.com/iap/docs/authentication-howto#authenticating_from_a_desktop_app.
       other_client_secret: The client secret used to obtain the auth codes and refresh tokens.
+      other_token: pass in token directly, it's used for cases better get token outside of SDK, e.x. GCP Cloud Functions.
```

Comment:
Is `other_token` an official term?

Reply:
no, just followed the style of upper field

Reply:
renamed to "existing_token" which I think you may like it.